tests/test_common.py

```diff
@@ -1,5 +1,238 @@
+from typing import Iterable
```

Comment:
My idea for test_common was to factor repetitions in tests (like a list of spaces that we want to test or environments) similar to https://github.com/Farama-Foundation/Gymnasium/blob/main/tests/testing_env.py

So good to have here the environments that we can reuse in multiple tests; for the test themselves I will keep the structure of the package as before (with `test_minari_storage.py`and `test_minari_dataset.py`)

To avoid confusion probably a better name of `test_common.py` is just `common.py`

Reply:
yeah that makes sense to avoid the test file name format, I'll make that change.

Reply:
done!